core/arch/arm/plat-imx/conf.mk

```diff
@@ -150,6 +150,7 @@ else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx8mm-flavorlist)))
 $(call force,CFG_MX8MM,y)
 $(call force,CFG_ARM64_core,y)
 CFG_IMX_UART ?= y
+CFG_IMX_I2C ?= y
```

Comment:
Is enabling the driver by default really needed?

Reply:
no not really. will remove